packages/gatsby-plugin-performance/cache-dir/find-path.js

```diff
@@ -0,0 +1,146 @@
+// import { pick } from "@gatsbyjs/reach-router/lib/utils"
```

Comment:
why do not remove all comments in this file?

Reply:
Just to make the diff a little bit easier when upgrading to newer versions of the framework

Reply:
@tlgimenes please then comment this at the top of the file or something, to avoid someone who doesn't know the reason for this commented-out code erase it in the future